packages/rs-dapi-client/src/transport/grpc.rs

```diff
@@ -97,6 +90,8 @@ macro_rules! impl_transport_request_grpc {
 
 // Link to each platform gRPC request what client and method to use:
 
+const STREAMING_TIMEOUT: Duration = Duration::from_secs(5 * 60);
```

Comment:
Isn't this a bit long?

Reply:
For proper using of stream it's fine. In future it's probably will be even without timeout when we implement PlatfromWallet